src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -437,6 +457,59 @@ impl DeviceManager {
 
         Err(anyhow!("ID are exhausted"))
     }
+
+    async fn try_update_device(&mut self, updated_config: &DeviceConfig) -> Result<()> {
+        let device_id = match updated_config {
+            DeviceConfig::ShareFsCfg(config) => {
+                // Try to find the sharefs device. If found, just return.
```

Comment:
The code is clear enough to understand that we're trying to find the device. Also, the "If found, just return" looks suspicious as the code is doing the opposite, i.e. continue if found, otherwise return an error.

Please, either put something informative that isn't obvious from _just_ reading the code or just drop this comment.

Reply:
I think the idea was "if found just use the device we found". I also think that the code tells it all quite clearly here and the comment can be dropped.

Reply:
fixed